theano/gpuarray/dnn_gi.c

```diff
@@ -13,8 +13,8 @@ if (PARAMS->choose_algo) {
 
 int reuse_algo;
 cudnnConvolutionBwdDataAlgo_t prev_algo;
-size_t prev_kern_dims[5] = {0};
-size_t prev_top_dims[5] = {0};
+size_t prev_kern_dims[5];
+size_t prev_top_dims[5];
```

Comment:
Ok. In that case, I will have to update the three files (dnn_fwd, dnn_gw, dnn_gi), as they have currently the same policy about these arrays.